packages/design-tokens/src/tokens/color.tokens.json

```diff
@@ -175,12 +175,18 @@
     "icon-map": {
       "value": "#4A63FC"
     },
+    "step-progress-background": {
+      "value": "#4A63FC"
+    },
     "step-progress-text": {
       "value": "#FFFFFF"
     },
     "map-background": {
       "value": "#102EE9"
     },
+    "footer-avatar-border": {
```

Comment:
Should it be footer-avatar-decoration since it's not a border on the CSS sense of things?